plugins/wacom-raw/fu-wacom-aes-device.c

```diff
@@ -253,6 +269,14 @@ fu_wacom_aes_device_write_firmware(FuDevice *device,
 						(gsize)i,
 						(gsize)chunks->len);
 	}
+
+	/* finalize the device */
+	if (!fu_wacom_aes_device_finalize_firmware(device, error)) {
+		return FALSE;
+	}
+
+	g_usleep(500 * 1000);
```

Comment:
what's this sleeping after the firmware is done writing for?  If it's real and not debugging, it needs a comment explaining it.

Reply:
I'll leave the comment for it.
The sleep is required for our AES firmwrae to get back to runtime mode.

Reply:
I think you might want to be using `fu_device_get_acquiesce_delay` then instead

Reply:
Thank you for your advice. 
I looked up a sample to use that function, but don't know how to set desired time on it.
Would you like to explain how I wait for 500msec with fu_device_get_acquiesce_delay?
Only I found an example was in fu-engine.c.

Reply:
I think the hardcoded delay is fine -- if the hardware cannot actually respond for that time. I think the acquiesce delay is only really useful for composite devices - it's certainly harder to explain for the single-device update. :)